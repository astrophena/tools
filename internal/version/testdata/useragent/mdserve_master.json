{
  "GoVersion": "go1.23.2",
  "Path": "go.astrophena.name/tools/cmd/mdserve",
  "Main": {
    "Path": "go.astrophena.name/tools",
    "Version": "v0.6.1-0.20241125133228-e92f88537561",
    "Sum": "h1:hI6t7EyqtgQokeq0eic7QPYUBM2lSQpW46F0v9+QwUc=",
    "Replace": null
  },
  "Deps": [
    {
      "Path": "github.com/benbjohnson/hashfs",
      "Version": "v0.2.2",
      "Sum": "h1:vFZtksphM5LcnMRFctj49jCUkCc7wp3NP6INyfjkse4=",
      "Replace": null
    },
    {
      "Path": "go.astrophena.name/base",
      "Version": "v0.1.3",
      "Sum": "h1:oER0DEURe50VI463pHtgq2ekLm/lsWiNTDg1hBjLqqU=",
      "Replace": null
    },
    {
      "Path": "golang.org/x/crypto",
      "Version": "v0.29.0",
      "Sum": "h1:L5SG1JTTXupVV3n6sUqMTeWbjAyfPwoda2DLX8J8FrQ=",
      "Replace": null
    },
    {
      "Path": "golang.org/x/net",
      "Version": "v0.27.0",
      "Sum": "h1:5K3Njcw06/l2y9vpGCSdcxWOYHOUk3dVNGDXN+FvAys=",
      "Replace": null
    },
    {
      "Path": "golang.org/x/text",
      "Version": "v0.20.0",
      "Sum": "h1:gK/Kv2otX8gz+wn7Rmb3vT96ZwuoxnQlY+HlJVj7Qug=",
      "Replace": null
    },
    {
      "Path": "rsc.io/markdown",
      "Version": "v0.0.0-20240717201619-868a055c40ae",
      "Sum": "h1:HcQakZADzQk1CHAKfBN7dNxRr/41J+CEEyUyFzbUa88=",
      "Replace": null
    }
  ],
  "Settings": [
    {
      "Key": "-buildmode",
      "Value": "exe"
    },
    {
      "Key": "-compiler",
      "Value": "gc"
    },
    {
      "Key": "-trimpath",
      "Value": "true"
    },
    {
      "Key": "CGO_ENABLED",
      "Value": "0"
    },
    {
      "Key": "GOARCH",
      "Value": "arm64"
    },
    {
      "Key": "GOOS",
      "Value": "android"
    },
    {
      "Key": "GOARM64",
      "Value": "v8.0"
    }
  ]
}