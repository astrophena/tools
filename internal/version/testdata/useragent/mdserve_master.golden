mdserve/v0.6.1-0.20241125133228-e92f88537561 (+https://astrophena.name/bleep-bloop)